team/index.html

```diff
@@ -2472,7 +2472,6 @@ <h1>Alumni</h1>
 						<span>Full Stack Developer</span>
 					</div>
 				</div>
-
```

Comment:
Revert this change

Reply:
I made those changes because nikit19 told me to remove the unnecessary blank line i had left. Can you be more specific?